buyer-api/scripts/test.sh

```diff
@@ -41,7 +41,7 @@ start_signing_service() {
 
   currentDir=`pwd`
   cd ../buyer-signing-service
-  npm start > /dev/null &
+  BUYER_PRIVATE_KEY=6ce09f3ea6dc337fa8c9f90ebc61d5d60e3da21b719d0d83f76dae875dc377e3 npm start > /dev/null &
```

Comment:
I can't come up with a better idea for this right now. Let's think how we can improve it.

Reply:
Just to keep it written somewhere, there are 3 cases for ganache
- You don't have it running, it will work ✔️ 
- You just ran `ganache-cli` with default options, it will work ✔️ 
- You executed `ganache-cli` with custom accounts/private keys, here it will fail because of the hardcoded `BUYER_PRIVATE_KEY` (unless you update the `test.sh` with the corresponding one). ❌